modules/mono/editor/GodotTools/GodotTools/Ides/Rider/RiderPathManager.cs

```diff
@@ -49,15 +49,16 @@ public static void Initialize()
                 if (!paths.Any())
                     return;
 
-                var newPath = paths.Last().Path;
+                string newPath = paths.Last().Path;
                 Globals.EditorDef(EditorPathSettingName, newPath);
                 editorSettings.SetSetting(EditorPathSettingName, newPath);
             }
         }
 
         public static bool IsExternalEditorSetToRider(EditorSettings editorSettings)
         {
-            return editorSettings.HasSetting(EditorPathSettingName) && IsRider((string)editorSettings.GetSetting(EditorPathSettingName));
+            bool hasSetting = editorSettings.HasSetting(EditorPathSettingName);
+            return hasSetting && IsRider((string)editorSettings.GetSetting(EditorPathSettingName));
```

Comment:
Why a new variable rather than a line break?

Reply:
Either way works, the line was too long as-is.